src/model/request/CeloMintErc721.ts

```diff
@@ -9,4 +11,10 @@ export class CeloMintErc721 extends MintErc721 {
   @IsNotEmpty()
   @IsIn([Currency.CELO, Currency.CUSD])
   public feeCurrency: Currency;
+  @IsOptional()
```

Comment:
this can be @IsNotEmpty(). When you use validateIf(), validations are not applied if the result is false. IF the result is true, validations are applied and this should not be empty when cashbackValues is present